### DISCUSSION TITLE
Swagger UI is not responsive

### DISCUSSION THREAD

**Discussion by iramakant1998** [swagger-api/swagger-ui#8342] [Q&A]
(https://user-images.githubusercontent.com/43852335/211888759-4923b1bd-8825-461d-87ff-2606e2077e7c.png)

Please take a look into this issue

**Comment by cihatsolak** [comment]
I've experienced this before. It happened when I updated the swagger packages in my project. For solution, check package versions. If necessary, downgrade the package version for a while.
